src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/services/WorkflowService.scala

```diff
@@ -166,7 +168,7 @@ class WorkflowServiceImpl(override val workflowRepository: WorkflowRepository,
           jobDefinitions = dagDefinitionJoined.jobDefinitions.filter(job => jobIds.contains(job.id))
         )
         for {
-          dagInstanceJoined <- jobTemplateService.resolveJobTemplate(dagDefinitionWithFilteredJobs, userName)
+          dagInstanceJoined <- dagInstanceService.createDagInstance(joinedWorkflow.dagDefinitionJoined, userName)
```

Comment:
Nope this will not work, point of this function was to run selected jobs. 
dagDefinitionWithFilteredJobs is not used.

Reply:
Omg, how could I overlook this 😅